optaplanner-core/src/test/java/org/optaplanner/core/api/solver/SolverManagerTest.java

```diff
@@ -49,11 +48,11 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertSame;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
-import static org.optaplanner.core.impl.testdata.util.PlannerAssert.assertSolutionInitialized;
+import static org.junit.Assert.*;
```

Comment:
Wouldn't it be better to import only the asserts used instead of using `org.junit.Assert.*`?

Reply:
We more-or-less have an exception for static imports for Assert and Mockito.
I'd love for us to just follow the quarkus code style and enforce it in the build just like they do.

But trying to change or synchronize **Code Style is a religious discussion**. Such discussions always spiral out of control, take a lot of energy and kill productivity.
So lets not discuss whether or not star static imports are a good thing. Let's discuss whether we "just want to follow quarkus's style" (or spring's) and be done with it. And when we want to investigate that effort.